docs/content/en/docs/troubleshooting/troubleshooting.md

```diff
@@ -248,6 +248,40 @@ kubectl logs <podname> -n <namespace> --kubeconfig=<kubeconfig>
 ....
 ```
 
+## Kubectl commands return dial tcp: i/o timeout
+
+If you are unable to run kubectl commands on a cluster due to timeout errors, then check if the server endpoint in the kubeconfig matches the control plane's endpoint in the infrastructure provider. If the endpoints do not match, you can ssh into the control plane node to gather logs instead. The kubelet logs can be obtained by running `journalctl -u kubelet.service --no-pager`. It may also be helpful to look at kube-vip logs, which can be found in the `/var/log/pods/kube-system_kube-vip-*` directory. 
+
+# Verify Cluster Certificates are valid
+
+If kubectl commands are not working, then it may also be helpful to verify that certificates on the etcd and control plane nodes have not expired. 
```

Comment:
what does it mean by `kubectl` commands are not working? is it same as above it's timing out? u should be more explicit describing the problem